### PR TITLE
Removal of ES dependency in arlas-rest

### PR DESCRIPTION
**Pull request by alainbodiguel**
Moving code to ExploreServices (not writing new) in order to remove dependency to ES in REST services

### CODE REVIEW

File: arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.utils;
+
+import javax.ws.rs.core.UriInfo;
+
+public class UriInfoUtil {
```
Comment (laurent-thiebaud-gisaia) [inline arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java]: I personally find the name may be misleading, I expect static methods from an Util class, whereas this is a wrapper.

File: arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java [hunk 2]
```diff
@@ -202,8 +214,154 @@ public ComputationResponse compute(MixedRequest request, CollectionReference col
         return computationResponse;
     }
 
+    public Hits getSearchHits(MixedRequest request, CollectionReference collectionReference, Boolean flat, UriInfo uriInfo, String method) throws ArlasException {
```
Comment (laurent-thiebaud-gisaia) [inline arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java]: I suppose you only moved the code and didn't change anything?

Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java]: Indeed, you can compare from the original REST services.

File: arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java (hunk 1 above)
Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java]: You're right. Would it make more sense to rename the class to UriInfoWrapper or reimplement it with static methods?

Comment (laurent-thiebaud-gisaia) [reply arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java]: I don't know how it is used, I feel like there will be more boilerplate code with static methods, but this is your choice :-)

Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java]: Ok. Renamed to UriInfoWrapper

**Pull request merged**